vortex/vortex-tools.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/bash
+printf "%s\n" "INFO: Running Pikdum's Tools for Vortex";
+printf "%s\n" "INFO: Checking for updates";
+PIKDUMSVERSIONNEW=$(curl https://raw.githubusercontent.com/SirStig/Steam-deck-tests/main/CHANGELOG.md);
+PIKDUMSVERSION=$(<~/.pikdum/steam-deck-master/CHANGELOG.md);
+if [ $PIKDUMSVERSION != $PIKDUMSVERSIONNEW ]; then
+    printf "%s\n" "UPDATE: Found new pikdum version. Downloading...";
+    ~/.pikdum/steam-deck-master/update.desktop
+else
+    printf "%s\n" "UPDATE: Pikdum's Tools is up to date!";
+fi;
+printf "%s\n" "INFO: Please select a tool from below by typing 1, 2, or 3:";
```

Comment:
**question:** How much of a pain is it to interact with this with only the virtual keyboard?

That's one of the reasons I made all the original shortcuts non-interactive, because I wanted to make sure it was as easy as possible to use on Steam Deck.

Reply:
Ah, well I think more in terms of Linux and steam deck. All you have to do is type 1 2 or 3. Which to me is easy. Do Steam + X to open the keyboard and type one.

Now for the vanilla launcher swapper they do have to type the entire game id.... but I didn't think using the keyboard a tiny bit for more control was an issue

Reply:
Could make the Vanilla launcher swapper also just go based off of 0, 1, 2, 3, etc... based off of the array index instead of game id

Reply:
Could the launcher swappers be global and non-interactive?